lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -5573,8 +5573,13 @@ namespace Js
     void JavascriptOperators::OP_InvalidateCachedScope(void* varEnv, int32 envIndex)
     {
         FrameDisplay *disp = (FrameDisplay*)varEnv;
-        RecyclableObject *objScope = RecyclableObject::FromVar(disp->GetItem(envIndex));
-        objScope->InvalidateCachedScope();
+        Var item = disp->GetItem(envIndex);
```

Comment:
Under what condition do we actually have a null ptr taking up space in the FrameDisplay?

Reply:
This used to be Null RecyclableObject object and RecyclableObject::InvalidateCachedScope is a no-op. 

LowerLdFrameDisplay copies the content of environment which can copy nullptr if we  did stackarg optimization.

Exprgen generate tons of such cases.

Reply:
That raises the question: why do we access the null slot in the FrameDisplay? You'd think we would only do that in the presence of eval, but if there's eval we shouldn't be doing stack args.